style/style.css

```diff
@@ -56,32 +56,26 @@
 
 /*________________________________________________________*/
 
-
 .general-footer {
   align-items: center;
   box-sizing: border-box;
-  background: #10367C;
+  background: #10367c;
   max-width: 1440px;
   height: 625px;
   align-items: center;
 }
 
-
 .footer-container {
-
   display: flex;
   justify-content: space-between;
   margin: 0 272px 64px 104px;
-
 }
 
 .all-colum {
```

Comment:
A cool solution is to assign a common style to the columns and each one separately! Well done!

Reply:
I advise you to set the width for all columns, for example 30% - it will be nice